advanced/readme_kiali.adoc

```diff
@@ -0,0 +1,183 @@
+= Observing Microservice meshes with Kiali
+
+At some point on the journey of everyone that is venturing on the world of
+microservices comes the need to actually visualize what is happening on the
+mesh. Questions like "who's connecting to who?", "how much of the traffic goes
+to each microservice?" are hard to answer because of how loosely tied
+microservices are, by definition.
+
+Those are the kinds of question that Kiali has the ability to answer, by giving
+you a big picture of the mesh, and showing the whole flow of your requests and
+data.
+
+== How does it work?
+
+Kiali taps into the data provided by Istio and Openshift to generate its
+visualizations. It fetches ingress data (such as request tracing with Jaeger),
+the listing and data of the services, health indexes, and so on.
+
+It runs as a service together with Istio, and does not require any changes to
+Istio or Openshift configuration (besides the ones required to install Istio).
+
+== How to install it?
+
+Those are the instructions to install the latest version of Kiali available. It
+requires Openshift and Istio to be configured and running correctly, so go do
+that now if you haven't yet.
+
+[source, bash]
+----
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml | \
+   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
+
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-secrets.yaml | \
+   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
+
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
+   IMAGE_NAME=kiali/kiali \
+   IMAGE_VERSION=latest \
+   NAMESPACE=istio-system \
+   VERSION_LABEL=master \
+   VERBOSE_MODE=4 envsubst | oc create -n istio-system -f -
+----
+
+This will take a while (usually up to one minute). After the service is
```

Comment:
Installing Kiali may take a minute or two.  You can use the following commands to see if the service is running: